cmd/subnetcmd/import.go

```diff
@@ -37,6 +43,24 @@ with the same name. To allow overwrites, provide the --force flag.`,
 		false,
 		"overwrite the existing configuration if one exists",
 	)
+	cmd.Flags().StringVar(
```

Comment:
probably we need to improve the Long description, and maybe add a comment at the start of the function. be clear that it either imports from a file or from apm

Reply:
Reworked the description